link/src/main/java/com/stripe/android/link/ui/inline/LinkInlineSignup.kt

```diff
@@ -154,37 +156,42 @@ internal fun LinkInlineSignup(
                         shape = MaterialTheme.linkShapes.medium
                     )
             ) {
-                Row(
+                Column(
```

Comment:
Can we achieve the same thing without the `Column`? I think this order of modifiers should work:

```kotlin
Row(
  modifier = Modifier
    .clip(MaterialTheme.linkShapes.medium)
    .clickable { toggleExpanded() }
    .padding(16.dp)
) {
  // …
}
```

Reply:
There are multiple items in the column. `AnimatedVisibility` below is in it.